### ISSUE TITLE
Running update_bus_stops deletes timetable data

### ISSUE THREAD

**Issue by jw35** [ijl20/tfc_web#188]
```
(tfc_web_venv) tfc_prod@tfc-app4:~/tfc_web/tfc_web$ psql tfcweb
psql (9.5.13)
Type "help" for help.

tfcweb=> select count(*) from transport_timetable;
  count
---------
 6946655
(1 row)

tfcweb=> \q
(tfc_web_venv) tfc_prod@tfc-app4:~/tfc_web/tfc_web$ ./manage.py update_bus_stops
(tfc_web_venv) tfc_prod@tfc-app4:~/tfc_web/tfc_web$ psql tfcweb
psql (9.5.13)
Type "help" for help.

tfcweb=> select count(*) from transport_timetable;
 count
-------
     0
(1 row)

tfcweb=> \q
```

Possibly because of FOREIGN KEY CONSTRAINTs in various of the timetable tables.

**Comment by jw35** [comment]
Will be resolved by #191 once master is deployed on smartcambridge.org

**Comment by jw35** [comment]
Master deployed
